Automattic/wp-calypso
# Component: Create a post stats card component

#### Proposed Changes

<img width="724" alt="image" src="https://user-images.githubusercontent.com/4044428/206592584-f28820d1-cf51-4ef8-bd74-96356db1018e.png">

<img width="504" alt="image" src="https://user-images.githubusercontent.com/4044428/206592450-7b67c4eb-49e5-4005-88bf-99839ef91dcd.png">

<img width="428" alt="image" src="https://user-images.githubusercontent.com/4044428/206822762-f523bf17-9fe9-431d-b948-2cae84be9b11.png">


Create a new component for showing post stats in a card. Note that the above screenshots were taken in an environment without TypeKit active and do not use the fonts from the design specifications.

#### Testing Instructions



* Spin up these changes on a local Storybook instance via `yarn workspace @automattic/components run storybook`.
* Take a look at the "Post Stats Card" stories and ensure they look reasonable.

#### Pre-merge Checklist





Related to #70852.
